src/bindings/c/include/openvino/c/ov_property.h

```diff
@@ -49,24 +39,114 @@ typedef enum {
                        //!< tasks. On the hybrid CPUs this option is default
 } ov_affinity_e;
 
+/**
+ * @enum ov_hint_priority_e
+ * @brief Enum to define possible priorities hints
+ */
+typedef enum {
+    LOW = 0,           //!<  Low priority
+    MEDIUM = 1,        //!<  Medium priority
+    HIGH = 2,          //!<  High priority
+    DEFAULT = MEDIUM,  //!<  Default priority is MEDIUM
+} ov_hint_priority_e;
+
+/**
+ * @enum ov_log_level_e
+ * @brief Enum to define possible log levels
+ */
+typedef enum {
+    NO = -1,      //!< disable any logging
+    ERR = 0,      //!< error events that might still allow the application to continue running
+    WARNING = 1,  //!< potentially harmful situations which may further lead to ERROR
+    INFO = 2,     //!< informational messages that display the progress of the application at coarse-grained level
+    DEBUG = 3,    //!< fine-grained events that are most useful to debug an application.
+    TRACE = 4,    //!< finer-grained informational events than the DEBUG
+} ov_log_level_e;
```

Comment:
I didn't find, where do we use these enums?

Reply:
It is the enumeration value of ov::hint::Priority property, if user pass a string, such as "WARNING", it cannot be recognized, so we should expose such enumeration to make user know how to set it.

Reply:
Could you show me the code where and how we are using it?

Reply:
yes, we can find enum property test case, which applies enum value replace string value

```
    const char* key_1 = ov_property_key_hint_performance_mode;
    ov_performance_mode_e mode = ov_performance_mode_e::THROUGHPUT;
    OV_ASSERT_OK(ov_core_set_property(core, device_name.c_str(), key_1, mode));
```

Reply:
The problem is that I don't see that you use enum `ov_hint_priority_e` or `ov_log_level_e` in tests or any other places.
Moreover if I remember right C++ support serialization/deserialization of enum values to string. Doesn't it work here?

Reply:
oh, I'm nor aware of such feature, let me check whether this serialization/deserialization can help.

Reply:
I think the serialization/deserialization is the different thing, which cannot help on this point.

BTW, the test cases only provides one of enum property test - ov_performance_mode_e::THROUGHPUT - if need more enum property tests, we can put more enum properties into it: (ov_performance_mode_e, ov_affinity_e, ov_hint_priority_e, ov_log_level_e), all of them will be provided to C user and passed to C property APIs as variadic arguments.

Reply:
If I remember right ov::Any supports serialization/deserialization to/from string for values.
@ilya-lavrenov Am I right?

Reply:
yes, you are right! It can convert number string to number, such as "2" -> 2, but it seems it cannot convert string of enum to enum value, for example:
"THROUGHPUT" -> ov::hint::PerformanceMode::THROUGHPUT (==2)
It cannot work.

Reply:
oh, I have made a mistake, we cannot convert enum string value to be int32_t value in ov::Any, but can convert to enum value:

```
        ov::Any test_value = "HYBRID_AWARE";
        std::string test_property_key="AFFINITY";
        std::cout << "test property = " << test_value.as<ov::Affinity>() << std::endl;
        ov::AnyMap test_property;
        test_property[test_property_key] = test_value;
        core->object->set_property(device_name, test_property); 
```

Reply:
So in this case I think we can remove enum specific code from property conversion and use strings instead of int.

Reply:
yes, I have submit commit to resolve it.